utils/getInvoices/index.ts

```diff
@@ -246,7 +246,7 @@ export const getElectricityInvoice = ({
       amount: +toRoundFixed(prevElectricity?.amount),
       lastAmount: +toRoundFixed(prevElectricity?.amount),
       price: +toRoundFixed(service.electricityPrice),
-      sum: 0,
+      sum: +toRoundFixed(service.electricityPrice),
```

Comment:
Let's calculate this one. Basically, for example. We have:

```
prevElectricity.amount = 10
service.electricityPrice = 3.5
```

Than we should have:

```
amount: prevElectricity.amount,
lastAmount: prevElectricity.amount,
price: service.electricityPrice,
sum: (this.amount - this.lastAmount) * this.price
```

And if everything is correct:

```
amount: 10,
lastAmount: 10,
price: 3.5,
sum: (10 - 10) * 3.5 = 0 * 3.5 = 0
```

So... Fresh created electricity will always have sum equal to 0. Change my ming or bring 0 sum back, please